app/src/androidTest/java/com/github/HumanLearning2021/HumanLearningApp/model/UniqueDatabaseManagementTest.kt

```diff
@@ -70,16 +68,4 @@ class UniqueDatabaseManagementTest {
             equalTo(oDbman.getCategories().map { cat -> cat.id })
         )
     }
-
-    @Ignore("functionality removed for the time being")
-    @Test
-    fun offlineDatabaseThrowsIfNotDownloaded() = runBlocking {
```

Comment:
- If there's no functionality to test, yeet
- This doesn't call into the DUT, instead it goes to an internal constructor of a different class. Irrelevant, therefore yeet

Reply:
ok